### ISSUE TITLE
[Bug?] Morphic tools and Tinker's Construct

### ISSUE THREAD

**Issue by Kriss8089** [TheCodex6824/ThaumicAugmentation#305]
Recently I've been experimenting with the morphic armor/tools since the modpack I'm playing does not have an aesthetic armor mod, and I wanted to see if it worked with Tinker's, as my new gear is mainly from that. I've found a few things:

1. Morphic armor via Construct's Armory works just fine, no issues there.
2. Morphic tools/weapons via Tinker's Construct have zero durability, are broken, and cannot be repaired in any way.

Is this a bug, or is Tinker's pretty much incompatible with TA's morphic tools?

**Comment by TheCodex6824** [comment]
I'm assuming the bottom right was supposed to say "normal weapon disguised as tinker's weapon". If that's the case, am I correct in saying that the issue is only when a tinker's weapon is disguised as a normal weapon? Also, not being repairable in an anvil is intentional due to some technical issues, but things like void metal and I suppose mending should still work, so if those don't as well that would be worth looking into.

**Comment by Kriss8089** [comment]
You are correct on the first thing lol, didn't notice that while putting this together. Also, the issue is when a tinker's weapon is disguised as anything else. It doesn't even register as anything in the tool forge, and even with the mending modifier, it does nothing (I think because mending only works when it has durability/not broken?). In the tool forge, it doesn't even register as anything by tinker's, so it can't be repaired that way. Even had a void metal tinker's sword get disguised as a regular void sword from thaumcraft, did not repair itself at all.
